File: DownstreamService/src/main/java/com/stackroute/downstreamservice/domain/BasicInfo.java
package com.stackroute.downstreamservice.domain;


import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class BasicInfo {
    private String dob;
    private String githubUrl;
    private String linkedinUrl;
    private String gender;
    private String contactNo;
}

File: Angular/src/app/info/basic-info/basic-info.ts
export class BasicInfo {
    constructor(private gender: string,
        private dob: string,
        private contactNo: string,
        private linkedinUrl: string,
        private githubUrl: string) {
        this.gender = gender;
        this.dob = dob;
        this.contactNo = contactNo;
        this.linkedinUrl = linkedinUrl;
        this.githubUrl = githubUrl;
    }
}
